src/Tests/Microsoft.NET.Sdk.WorkloadManifestReader.Tests/SdkDirectoryWorkloadManifestProviderTests.cs

```diff
@@ -526,36 +526,6 @@ var sdkDirectoryWorkloadManifestProvider
                 .BeEquivalentTo("ios: 11.0.2/8.0.100");
         }
 
-        [Fact]
-        public void ItFailsIfWorkloadSetFromGlobalJsonIsNotInstalled()
```

Comment:
I think we should continue to fail builds if the version specified in global.json isn't available, so these two tests should be kept.